packages/manager/fragments/navbar/src/language/button.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+// eslint-disable-next-line react/display-name
+const DropdownLanguageToggle = React.forwardRef(
+  // eslint-disable-next-line react/prop-types
```

Comment:
To avoid `forwardRef render functions do not support propTypes or defaultProps.`